notebooks/04.02-Simple-Scatter-Plots.ipynb

```diff
@@ -37,10 +37,8 @@
   },
```

Comment:
<p>n the previous section we looked at&nbsp;<code>plt.plot</code>/<code>ax.plot</code>&nbsp;to produce line plots. It turns out that this same function can produce scatter plots as well:n the previous section we looked at&nbsp;<code>plt.plot</code>/<code>ax.plot</code>&nbsp;to produce line plots. It turns out that this same function can produce scatter plots as well:</p>
<br/>

 _Reply via <a href='http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>a</p>